test/test_basic.py

```diff
@@ -18,6 +18,12 @@ async def askpass(self, msg, hint):
 
 
 class TestBasic(unittest.IsolatedAsyncioTestCase):
+    @classmethod
+    def setUpClass(kls):
+        # git does not track file permissions, and SSH fails on group/world readability
+        os.chmod('test/id_rsa', 0o600)
```

Comment:
I don't care for this — it's effectively modifying the source tree.  Isn't there a way we can tell SSH to be less strict about this?  If that's the way it's gotta be then we should create temporary copies of these elsewhere.

Reply:
It makes interactive debugging a bit more awkward, but then one can still chmod the original files. Changed to a temp dir.